src/WooCommerce.php

```diff
@@ -380,6 +387,7 @@ public static function woocommerce_save_product_variation($post_id, $loop) {
       return;
     }
     $variation_id = $_POST['variable_post_id'][$loop];
+    $gtin_error = false;
```

Comment:
We are following [Drupal Code Standards for constant naming](https://www.drupal.org/docs/develop/standards/coding-standards#s-constants). So `true` should be uppercase.